build_docs.pl

```diff
@@ -164,18 +164,17 @@ sub build_all {
     ensure_creds_cache_enabled() || enable_creds_cache() || exit(1);
     check_github_authed();
 
-    my $repos_dir = init_repos();
+    my ($repos_dir, $temp_dir, $target_repo, $target_repo_checkout) = init_repos();
```

Comment:
I move the `$temp_dir` clean and recreation into `init_repos` because I need to clone the target repo into it.